gems/manageiq_foreman/manageiq_foreman.gemspec

```diff
@@ -26,5 +26,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "webmock"
   spec.add_development_dependency "rspec"
 
+  spec.add_development_dependency "coveralls"
```

Comment:
I left it out of here because its use case is so narrow -- it makes little practical difference, but it seemed more elegant to relegate it to [the Gemfile](https://github.com/bdunne/manageiq/blob/only_push_to_coveralls_upstream/gems/manageiq_foreman/Gemfile). At least that way we wouldn't be publishing it as an official dev dep on rubygems.org (if this gem were published).